brave/browser/brave_browser_context.cc

```diff
@@ -353,6 +379,11 @@ BraveBrowserContext::CreateZoomLevelDelegate(const base::FilePath& partition_pat
       ui_zoom::ZoomEventManager::GetForBrowserContext(this)->GetWeakPtr()));
 }
 
+scoped_refptr<autofill::AutofillWebDataService>
+BraveBrowserContext::GetAutofillWebdataService() {
```

Comment:
along with the comment above, I think this should always return the AutofillWebDataService created by the regular browser context. The autofill driver will not write autofill history to the database when it is used with an incognito profile